channels/ae.m3u

```diff
@@ -123,21 +123,21 @@ https://shls-mbcplusdrama-prod-dub.shahid.net/out/v1/97ca0ce6fc6142f4b14c0a694af
 https://admdn2.cdn.mangomolo.com/nagtv/smil:nagtv.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="NoorDubai.ae" tvg-name="Noor Dubai" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://i.imgur.com/FP2yHkl.png" group-title="General",Noor Dubai (576p)
 https://dmiffthvll.cdn.mangomolo.com/noordubaitv/smil:noordubaitv.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="PeaceTVBangla.ae" tvg-name="Peace TV Bangla" tvg-country="BD;IN" tvg-language="Bengali" tvg-logo="https://i.imgur.com/ibUfEGr.png" group-title="Religious",Peace TV Bangla [Offline]
+#EXTINF:-1 tvg-id="PeaceTVBangla.ae" tvg-name="PeaceTV Bangla" tvg-country="BD;IN" tvg-language="Bengali" tvg-logo="https://i.imgur.com/ibUfEGr.png" group-title="Religious",PeaceTV Bangla [Offline]
 https://api.visionip.tv/live/ASHTTP/peacetv-peacetv-peacetv-bangla-hsslive-25f-16x9-MB/playlist.m3u8
-#EXTINF:-1 tvg-id="PeaceTVChinese.ae" tvg-name="Peace TV Chinese" tvg-country="CN" tvg-language="Chinese" tvg-logo="https://i.imgur.com/ibUfEGr.png" group-title="Religious",Peace TV Chinese [Offline]
+#EXTINF:-1 tvg-id="PeaceTVChinese.ae" tvg-name="PeaceTV Chinese" tvg-country="CN" tvg-language="Chinese" tvg-logo="https://i.imgur.com/ibUfEGr.png" group-title="Religious",PeaceTV Chinese [Offline]
 https://api.visionip.tv/live/ASHTTP/peacetv-peacetv-peacetv-china-hsslive-25f-16x9-MB/playlist.m3u8
```

Comment:
From https://peacetvnetwork.visionip.tv/live : PEACETV SERVICE IS TEMPORARLY UNAVAILABLE

Reply:
Are you suggesting that we remove the links from the playlist altogether?

Reply:
Well for the time being, better keep them as offline, in case they come back later.